ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableDataProvider.kt

```diff
@@ -46,7 +46,7 @@ internal class VariableDataProvider(
     }
 
     fun productName(rcPackage: Package): String {
-        return rcPackage.product.title
+        return rcPackage.product.name
```

Comment:
This is a behavior change, so the product names don't include the app name.

Reply:
But also a bug fix :)

Reply:
Actually what do you think of putting this in a separate PR so we can cool out that we fixed the variable?

Reply:
Yeah, I can do that, will need to update a couple tests but that's ok, will just put them back in that PR 👍

Reply:
Separated in https://github.com/RevenueCat/purchases-android/pull/1569